### ISSUE TITLE
达到的最好成绩。

### ISSUE THREAD

**Issue by fenggenb** [yuanqing811/ISIC2018#2]
你好，请问一下你们使用新划分的数据集最高能达到多少？
我们去了10%的训练集作为验证集，最后验证集上的acc达到了90%左右，balanced_acc只有80%左右。

**Comment by yuanqing811** [comment]
> Hello, how much can you achieve using the newly-divided data set?
We went to the 10% training set as the verification set, and the final verification set had an acc that reached about 90%, and balanced_acc was only about 80%.

Hi, that sounds similar to the numbers we are getting. We decided to use 20% dataset as verification set since a small size can possibly influence metric of balanced accuracy significantly.  Categories DF and VASC in particularly seems to have a small size.

**Comment by fenggenb** [comment]
Thank you for the reply. 
You team wrote great code.

**Comment by yuanqing811** [comment]
Validation split does NOT yet take into account lesion ID. We will fix that ASAP.

**Comment by yuanqing811** [comment]
> Does your new validation set take into account the lesson in?
Validation split now takes into account lesion IDs. Thank you for bringing it to our attention. Upgraded balanced accuracy is now 68.5% (decreased from 76% without taking into account lesion IDs).

**Comment by fenggenb** [comment]
Hi, our validation indeed set take into account the lesson ID.
We just classify each class into four types(single, serial, histopathology, dermoscopy), and split 10% of these types into validation set. I hope this can help you.Because We did get a better result.

**Comment by fenggenb** [comment]
I think you don't need to predict the lesion types of the test images. The metadata provided just let us divide a better validation set, in my opinion, test data is also produced by diagnosis type. What' s more, the metadata prevent us from splitting one diagnosis confirmation type into validation set, you know, if did these its predictions will be terrible, because train set will lose information with this diagnosis confirmation type. The ISIC also said it can help us split train/validation set, but I think your understanding maybe helpful for prediction.
My English is very poor, I was almost crazy due to the paper. Please let me know your opinion.
